subsys/bluetooth/mesh/adv_legacy.c

```diff
@@ -243,7 +248,11 @@ int bt_mesh_adv_disable(void)
 {
 	int err;
```

Comment:
btw. function just ignores returned thread status and always notifies caller about success. Doesn't seem correct.